release-tools/cloudbuild.yaml

```diff
@@ -27,7 +27,7 @@ steps:
   # The image must contain bash and curl. Ideally it should also contain
   # the desired version of Go (currently defined in release-tools/prow.sh),
   # but that just speeds up the build and is not required.
-  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20210331-c732583'
+  - name: 'gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211008-60e346af'
```

Comment:
This change was already merged into csi-release-tools in https://github.com/kubernetes-csi/csi-release-tools/pull/175/files

Someone just needs to update `release-tools` in this repo.

Reply:
I created https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/pull/163 for it.